extension.json

```diff
@@ -349,6 +349,36 @@
       },
       "libPath": "dist/lib/actions/redirectWithIdentity/index.js",
       "viewPath": "actions/redirectWithIdentity.html"
+    },
+    {
+      "displayName": "Apply response",
+      "name": "apply-response",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "type": "object",
+        "properties": {
+          "instanceName": {
+            "type": "string",
+            "minLength": 1
+          },
```

Comment:
should we add a `viewName` too? That is for the use case when the page loads on a SPA view. Currently WEB SDK detects if page loads on a SPA view and applies both the `__view__` and SPA view propositions.